cpp/src/Ice/LoggerAdminI.cpp

```diff
@@ -113,7 +113,7 @@ namespace
         virtual std::string getPrefix();
         virtual LoggerPtr cloneWithPrefix(const std::string&);
 
-        virtual std::shared_ptr<Ice::Object> getFacet() const;
+        virtual ObjectPtr getFacet() const;
```

Comment:
Seems like we can make `LoggerAdminLoggerI` class final.

Reply:
Fixed.